CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 
 
```

Comment:
nit: there's an extra here. Can fix that now or in the next release note. Other than that, LGTM 👍